cmd/format-config-v1.go

```diff
@@ -916,10 +1037,10 @@ func initFormatXL(storageDisks []StorageAPI) (err error) {
 		}
 		// Allocate format config.
 		formats[index] = &formatConfigV1{
-			Version: "1",
-			Format:  "xl",
+			Version: formatFileCurrent,
+			Format:  formatBackendXL,
 			XL: &xlFormat{
-				Version: "1",
+				Version: xlFormatBackendCurrent,
```

Comment:
1. `formatFileCurrent` and `xlFormatBackendCurrent` need to be `formatFileV1` and `xlFormatBackendV1` respectively for `formatConfigV1`.
2. You could have a separate `newXLFormatV1()` and use it here.